### PR TITLE
AI Takeover

### PR DESCRIPTION
**Pull request by jbradberry**
Adds the ability for the AI to take over for a human player if they walk away.

### CODE REVIEW

**Comment by jbradberry** [comment]
I'm also going to unbreak local human vs human play on this PR, whatever is going on with that.

File: tictactoe/t3backend/views.py [hunk 1]
```diff
@@ -49,23 +49,29 @@ class GameDetailAPIView(generics.RetrieveUpdateAPIView):
...
-
-        state = b.play(json.loads(serializer.instance.state), play)
-        jsonstate = json.dumps(state)
-
-        game = serializer.save(
-            state=jsonstate, last_play=json.dumps(play),
-            winner=b.winner([state])
-        )
+        if serializer.validated_data['play'] == 'timeout':
+            # The AI permanently takes over for a player when they walk away.
```
Comment (ironfroggy) [inline tictactoe/t3backend/views.py]: The AI timeout is intended to be per-move, and not permanent.

Comment (jbradberry) [inline tictactoe/t3backend/views.py]: Ok, it's easy enough to fix.

**Comment by ironfroggy** [comment]
+1

**Pull request merged**